sdk/lib/rtl/compress.c

```diff
@@ -59,7 +59,10 @@ static PUCHAR lznt1_decompress_chunk(UCHAR *dst, ULONG dst_size, UCHAR *src, ULO
 
                 /* find length / displacement bits */
                 for (displacement_bits = 12; displacement_bits > 4; displacement_bits--)
-                    if ((1 << (displacement_bits - 1)) < dst_cur - dst) break;
+                {
```

Comment:
This appears to be unmodified Wine code, so please keep it in sync.

Reply:
Submitted to Wine, stalled there...

Reply:
It's not a bug, it's a false positive. We can silence it with a compiler option. Please revert this part.